Practise_ClassWork/FridgeEvent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practise_ClassWork
+{
+    public class FridgeEvent:EventArgs
```

Comment:
Please align the code.

Reply:
Customized EventArgs should store state and the information about event has happened.

Reply:
-3 to custom EventArgs.